src/main/java/com/spartronics4915/frc2020/commands/IndexerCommands.java

```diff
@@ -376,9 +385,5 @@ public LoadToLauncher(int ballsToShoot)
             );
         }
 
-        public LoadToLauncher()
-        {
-            this(1);
```

Comment:
What’s the point of this change?

Reply:
Someone was calling the constructor with no argument.  Not clear what the intent was, but elsewhere someone was doing something related to this my my inference.